pages/learn/manage/device-statuses.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Device statuses
+excerpt: Determine the status and connectivity of a device
+---
+
+# Device statuses
+
+The application overview page shows the status of each device in the application.
+
+<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >
+
+Each device can have one of the following statuses:
+
+| Status            | Description                                                                                  |
+|-------------------|----------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to either the VPN or has recent API communications.    |
+| Configuring       | The device is applying a configuration update.                                               |
+| Updating          | The device is updating to a new application release.                                         |
+| Offline           | The device is offline and is not connected to VPN and has not any recent API communications. |
+| Post Provisioning | The device has been [provisioned][provisioned] but has not yet come online.                  |
```

Comment:
Correct :+1: 
To provide the full context: (for flasher device types) the flasher sets the device to this state, right before it issues a device restart.

See: https://github.com/balena-os/meta-balena/blob/006bc19161053147f47fa12e4cc063da5011215a/meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher#L144